solc/CommandLineInterface.cpp

```diff
@@ -121,8 +125,10 @@ static string const g_argFormal = g_strFormal;
 static string const g_argGas = g_strGas;
 static string const g_argHelp = g_strHelp;
 static string const g_argInputFile = g_strInputFile;
+static string const g_argJulia = "julia";
```

Comment:
I have absolutely no idea why we started introducing two global string variables for the same thing.